### ISSUE TITLE
After orientating the screen from portrait to landscape the data on screen is getting overlapped. 

### ISSUE THREAD

**Issue by SuryacharanM** [dotnet/maui#8607]
GitHubTags:
#A11ySev2; #.NET MAUI; #AILimited; #A11yTCS; #Android; #AndroidPhone; #BM_.NETMAUI_Android_Jul2022; #WCAG1.3.4;

### Description

After orientating the screen from portrait to landscape the data on screen is getting overlapped. 

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. "Home Screen" will open.
3. Observe the issue.

https://user-images.githubusercontent.com/86710788/177965473-9a3766e5-d35c-487b-8976-93483fed24e8.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Relevant log output

```shell
After orientating the screen from portrait to landscape the data on screen should not get overlapped.
```

**Comment by jsuarezruiz** [comment]
Looking at the video, it seems that you are using a CollectionView? Could you attach a small example where reproduce the issue or paste a piece of code?

**Comment by jsuarezruiz** [comment]
Oh, I noticed is the Recipe sample, right?

**Comment by merriemcgaw** [comment]
@SuryacharanM - please validate this bug so it can be closed ASAP.

**Comment by keerthiduvvuri** [comment]
Hi @merriemcgaw i have verified this bug. The issue is fixed.
I cannot close the bug, because this bug was logged by another tester. Can you close the bug from you'r end. Thanks.

https://user-images.githubusercontent.com/98728920/188080454-d8ec222e-2ca1-4f4a-97e5-21199166dcaa.mp4

**Comment by merriemcgaw** [comment]
@rachelkang will you do the honors 😄 ?

**Comment by rachelkang** [comment]
gladly - thanks all! :)
